tests/test_cola/test_login.py

```diff
@@ -28,18 +28,24 @@ def test_login(cola_client, cola_cognito_user_pool_jwk):
 
 
 @pytest.mark.django_db
-def test_logout(alice, cola_client):
+def test_logout(alice, cola_client, settings):
```

Comment:
```suggestion
def test_logout(alice, cola_client):
```
dont need the settings?

Reply:
We need them to get the cookie name